pytype/tools/analyze_project/pytype_runner.py

```diff
@@ -46,7 +47,8 @@ def _get_executable(binary, module=None):
               path_utils.abspath(path_utils.dirname(custom_bin)),
               'pytype-single')
       ])
-  if sys.executable is not None:
+  importable = importlib.util.find_spec(module or binary)
+  if sys.executable is not None and importable is not None:
```

Comment:
this can be just `... and importable` since `find_spec` does not return anything else that evaluates to `False`

Reply:
i prefer the two-liner, the variable name `importable` does add a nice bit of readability.